eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -28,98 +31,48 @@ parameters:
-        MacOs:
-          OSVmImage: "macOS-10.15"
-          TestTargetFramework: netcoreapp2.1
-    pool:
-      vmImage: "$(OSVmImage)"
-
-    steps:
-      - template: /eng/common/pipelines/templates/steps/bypass-local-dns.yml
-      
-      - ${{ parameters.PreSteps }}
-
-      - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
-
-      - task: UseDotNet@2
-        displayName: "Use .NET Core runtime $(DotNetCoreRuntimeVersion)"
-        inputs:
-          packageType: runtime
-          version: "$(DotNetCoreRuntimeVersion)"
-
-      - task: UseDotNet@2
-        displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
-        inputs:
-          packageType: sdk
-          version: "$(DotNetCoreSDKVersion)"
-
-      - template: /eng/common/TestResources/deploy-test-resources.yml
-        parameters:
-          Location: $(Location)
-          ServiceDirectory: '${{ parameters.ServiceDirectory }}'
-          SubscriptionConfiguration: $(SubscriptionConfiguration)
-
-      - script: >
-          dotnet test eng/service.proj
-          --framework $(TestTargetFramework)
-          --filter "TestCategory!=Manually"
-          --logger "trx"
-          --logger:"console;verbosity=normal"
-          /p:ServiceDirectory=${{ parameters.ServiceDirectory }}
-          /p:IncludeSrc=false /p:IncludeSamples=false
-          /p:BuildInParallel=${{ parameters.BuildInParallel }}
-          $(ConvertToProjectReferenceOption)
-
-        displayName: "Build & Test (all tests for $(TestTargetFramework))"
-        env:
-          DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
-          DOTNET_CLI_TELEMETRY_OPTOUT: 1
-          DOTNET_MULTILEVEL_LOOKUP: 0
-          AZURE_TEST_MODE: "None"
-          ${{ insert }}: ${{ parameters.EnvVars }}
-
-      - template: /eng/common/TestResources/remove-test-resources.yml
-        parameters:
-          ServiceDirectory: '${{ parameters.ServiceDirectory }}'
-          SubscriptionConfiguration: $(SubscriptionConfiguration)
-
-      - task: PublishTestResults@2
-        condition: always()
-        displayName: "Publish Results ($(TestTargetFramework))"
-        inputs:
-          testResultsFiles: "**/*.trx"
-          testRunTitle: "$(OSName) $(TestTargetFramework)"
-          testResultsFormat: "VSTest"
-          mergeTestResults: true
-
-      - ${{ parameters.PostSteps }}
+- template: archetype-sdk-tests-jobs.yml
+  parameters:
+    PreSteps: ${{ parameters.PreSteps }}
+    PostSteps: ${{ parameters.PostSteps }}
+    EnvVars: ${{ parameters.EnvVars }}
+    MaxParallel: ${{ parameters.MaxParallel }}
+    BuildInParallel: ${{ parameters.BuildInParallel }}
+    TimeoutInMinutes: ${{ parameters.TimeoutInMinutes }}
+    Location: ${{ parameters.Location }}
+    SubscriptionConfiguration: ${{ parameters.SubscriptionConfiguration }}
+    ServiceDirectory: ${{ parameters.ServiceDirectory }}
+    platforms:
+      - DisplayName: "Test on Linux"
+        OSVmImage: "ubuntu-18.04"
+        TestMode: None
```

Comment:
You should be able to set TestMode to None as default so you don't have to repeat.

Reply:
How do I do it if `platforms` is an array parameter?

Reply:
I left behind a comment where I think you can do this when setting the test mode.